### DISCUSSION TITLE
Does supabase's FDW with Redis require the redis connection to be TLS?

### DISCUSSION THREAD

**Discussion by harrisrobin** [supabase/supabase#30672] [Questions] [answered]
For some reason, I am unable to connect a redis connection if TLS is enabled so my question is, does the redis foreign data wrapper not work with TLS connections?

To be clear, when i try to use a TLS enabled redis connection i get:

```
can't connect with TLS, the feature is not enabled- InvalidClientConfig
```

**Comment by burmecia** [comment, marked as answer]
The wrappers v0.4.4 [added TLS support for Redis FDW](https://github.com/supabase/wrappers/releases/tag/v0.4.4), which is already available on latest Supabase platform.
